packages/manager/modules/pci/src/projects/project/notebooks/components/delete-notebook/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,8 @@
+{
+  "pci_notebook_delete_notebook_title": "Suppression du notebook",
```

Comment:
"notebook" is not a french word, i don't know if it's intended to keep it in english

Reply:
We dont have the content as yet. Sorry for not mentioning it earlier